### PR TITLE
Global style

### PR DESCRIPTION
**Pull request by mmyczkowski19**
add font Lato

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,21 @@
 import React from "react";
-import MainNav from "./components/main-nav";
-import ContactForm from "./components/contact-form";
+import MainNav from "./components/nav/main-nav";
+import ContactForm from "./components/contact-form/contact-form";
+import Header from "./components/header/header";
+import Welcome from "./components/welome/welcome";
+import styled from "styled-components";
 
+const MainStyle = styled.div`
```
Comment (chochstrasser) [inline src/App.js]: lets move this into its own file and then import it

File: src/components/header/header.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { StyledHeader, StyledContainer, Tel, Logo } from "./styled-header";
+const Header = () => {
+  return (
+    <StyledHeader>
+      <StyledContainer>
+        <Logo>
+          <b>Paint</b>Utah<b>Curbs</b>
+        </Logo>
+        <Tel>TEL: +1(435)-32332-34</Tel>
```
Comment (chochstrasser) [inline src/components/header/header.js]: who's number is this?

File: src/components/nav/main-nav.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ const MainNav = () => {
     <StyledNav>
       <NavList>
         <NavItem>
-          <NavLink href="#1">contact icon</NavLink>
+          <NavLink href="#1">Contact</NavLink>
```
Comment (chochstrasser) [inline src/components/nav/main-nav.js]: we will need to do some icon here with font-awesome... in the meantime this is good.

File: src/components/welome/welcome.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    <StyledWelcome>
+      <StyledContainer>
+        <StyledImg src="https://s3-us-west-2.amazonaws.com/www.paintutahcurbs.com/54.jpg" />
+        <StyledText>
+          <StyledWelcomeH1>Welcome</StyledWelcomeH1>
+          <StyledWelcomeText>
+            Having your curb address painting be clearly visible helps Police,
+            Fire, Ambulance Services & other Guests find your home, especially
+            at night when addresses can be hard to find. Statistics show that
+            78% of homes are inadequately numbered for night use, and that's
```
Comment (chochstrasser) [inline src/components/welome/welcome.js]: is this true stats ?

File: src/components/header/styled-header.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+import styled from "styled-components";
```
Comment (chochstrasser) [inline src/components/header/styled-header.js]: use header-styles.js instead of styled-header.js

File: src/components/welome/welcome.js (hunk 4 above)
Comment (mmyczkowski19) [reply src/components/welome/welcome.js]: I guess yes. I found this info on your competitor’s website from another state:-)
Please re-edit this copy. I just copied and pasted.

File: src/components/header/styled-header.js (hunk 5 above)
Comment (mmyczkowski19) [reply src/components/header/styled-header.js]: Ok.

File: src/components/header/header.js (hunk 2 above)
Comment (mmyczkowski19) [reply src/components/header/header.js]: I made up this number as a dummy phone number

File: src/App.js (hunk 1 above)
Comment (mmyczkowski19) [reply src/App.js]: Ok. I will

**Pull request merged**